src/late/npm-on-ipfs/package.js

```diff
@@ -0,0 +1,46 @@
+import { logger } from '../../utils'
+import util from 'util'
+
+const execFile = util.promisify(require('child_process').execFile)
+
+export async function update () {
+  try {
+    logger.info('[npm on ipfs] ipfs-npm: checking if outdated')
+
+    // NOTE: might fail on older NPM (< 6.9.1) versions
+    // https://github.com/npm/cli/pull/173
+    const { stdout } = await execFile('npm', ['outdated', '-g'])
+
+    if (stdout.indexOf('ipfs-npm') === -1) {
+      logger.info('[npm on ipfs] ipfs-npm: is up to date')
+      return
+    }
+  } catch (e) {
+    logger.error('[npm on ipfs] ipfs-npm: could not check if up to date %v', e)
```

Comment:
Maybe throw? we shouldn't let it fall through the next log line.

Reply:
My idea here was to ignore this error and update it anyways. It will fail for most people due to a bug on npm, which was not yet released: https://github.com/ipfs-shipyard/ipfs-desktop/blob/feat/npm-on-ipfs/src/late/npm-on-ipfs/package.js#L10-L11